docs/Inspection-of-yum-Repositories.md

```diff
@@ -42,10 +43,11 @@ You can find an example script below in the References section.
 * Disadvantages:
   * Hard to parse
   * Disabled repos missing
-  * Variables `$releasever`, `$basearch` in repo files cause raise questions:
+  * Variables `$releasever`, `$basearch` in repo files raise questions:
     * What about the variables used in repo files?
     * Should we store them or the expanded value?
     * Where do the values of the variables come from, and should we capture that as well?
```

Comment:
It would be great to have some answers to these questions, so we can decide how to deal with them.